tests/AddFeaturePolicyHeadersTest.php

```diff
@@ -57,7 +57,7 @@ public function configure(): void
             }
         };
 
-        config(['feature-policy.policy' => get_class($policy)]);
+        config()->set('feature-policy.policy', $policy::class);
```

Comment:
the `$policy::class` is a simpler way of getting the class, the `config()->set(..)` is (I think) more in line with the common way these statements are written.